rviz_default_plugins/src/rviz_default_plugins/displays/pointcloud/point_cloud_to_point_cloud2.cpp

```diff
@@ -64,22 +64,5 @@ sensor_msgs::msg::PointCloud2::ConstSharedPtr rviz_default_plugins::convertPoint
   output->is_bigendian = false;
   output->is_dense = false;
 
-  // Copy the data points
-  auto floatData = reinterpret_cast<float *>(output->data.data());
```

Comment:
This is not unused code; it is editing the output array.  The same goes for all of the blocks below.